templates/pimcore5/readme.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/README.md b/README.md
```

Comment:
The readme file should be just a standalone file that totally replaces the Pimcore one.  Making it a patch just complicates things.

Reply:
Done, I pushed changes for this.